File: uri/c/1005.c
#include <stdio.h>

#define WEIGHT_GRADE_A 3.5
#define WEIGHT_GRADE_B 7.5

int main() {

	double A, B, MEDIA;
	scanf("%lf %lf", &A, &B);

	MEDIA = ((A * WEIGHT_GRADE_A) + (B * WEIGHT_GRADE_B))/11;

	printf("MEDIA = %.5lf\n", MEDIA);

	return 0;

}

File: uri/go/1005.go
package main

import (
	"fmt"
)

func main() {

	var A float64
	var B float64

	fmt.Scanf("%f", &A)
	fmt.Scanf("%f", &B)

	media := (0.35 * A + 0.75 * B) / 1.10

	fmt.Printf("MEDIA = %.5f\n", media)

}